tools/run-live-validation.ps1

```
@@ -2,7 +2,12 @@ param(
     [string]$Configuration = "Release",
     [string]$ResultsDirectory = "TestResults",
     [string]$ProfileRoot = "profiles/default",
-    [switch]$NoBuild = $true
+    [switch]$NoBuild = $true,
```

PSScriptAnalyzer (1.23.0)

[warning] File 'run-live-validation.ps1' has a switch parameter default to true.

(PSAvoidDefaultValueSwitchParameter)